pathcore/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .feature_pathway_overrepresentation import \
+    pathway_enrichment_no_overlap_correction, \
+    pathway_enrichment_with_overlap_correction, \
+    single_side_pathway_enrichment
+from .network import CoNetwork
+from .network_permutation_test import aggregate_permuted_network, \
+        network_edges_permutation_test
+
+__all__ = ["CoNetwork",
```

Comment:
My understanding is that the only purpose of `__all__` is to enable `from pathcore import *` ([doc](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package)). Unless this API functionality is required, I'd discourage it. Import all is a [bad practice](https://stackoverflow.com/a/2386740/4651668) and something I'd choose not to support.